src/WikiService.Business/Commands/Article/Edit/EditArticleRequestValidator.cs

```diff
@@ -99,7 +106,45 @@ private async Task<bool> DoesExistAsync(Guid rubricId)
       _provider = provider;
 
       RuleForEach(x => x.Item2.Operations)
-        .CustomAsync(async (x, context, _) => await HandleInternalPropertyValidationAsync(x, context));
+        .CustomAsync(async (x, context, _) => await HandleInternalPropertyValidationAsync(x, context));
+
+      When(x => x.Item2.Operations.Any(o =>
+        (o.path.EndsWith(nameof(EditRubricRequest.ParentId), StringComparison.OrdinalIgnoreCase))
+          || (o.path.EndsWith(nameof(EditRubricRequest.IsActive), StringComparison.OrdinalIgnoreCase))),
+        () =>
+        {
+          RuleFor(x => x)
+            .MustAsync(async (x, _) =>
+            {
+              Guid? _currentRybricId = x.Item1.RubricId;
+              bool _currentIsActive = x.Item1.IsActive;
+
+              foreach (Operation<EditArticleRequest> item in x.Item2.Operations)
+              {
+                if (item.path.EndsWith(nameof(EditRubricRequest.ParentId), StringComparison.OrdinalIgnoreCase)
+                  && (Guid.TryParse(item.value.ToString(), out Guid parentId) || item.value is null))
+                {
+                  _currentRybricId = item.value is null
+                    ? null
+                    : parentId;
+                }
+                else if (item.path.EndsWith(nameof(EditRubricRequest.IsActive), StringComparison.OrdinalIgnoreCase)
+                  && bool.TryParse(item.value?.ToString(), out bool _))
+                {
+                  _currentIsActive = (bool)item.value;

```

Comment:
lines 132-134
it would not work?
...&& bool.TryParse(item.value?.ToString(), out bool isActive))
{
_currentIsActive = isActive;

Reply:
done